### PR TITLE
Homescreen blog section

### PR DESCRIPTION
**Pull request by jfidiles**

### CODE REVIEW

File: _sass/main/_home.scss [hunk 1]
```diff
@@ -169,4 +169,55 @@ $content-max-width-medium: 620px;
...
+.Home-blogDate:after {
+  $size: 2px;
+
+  content: " ";
+  width: $size;
+  height: $size;
+  display: inline-block;
+  border-radius: 3px;
+  background-color: $color-grey-light;
+  position: relative;
```
Comment (artisologic) [inline _sass/main/_home.scss]: Would a negative margin work for this?

File: _sass/main/_home.scss [hunk 2]
```diff
@@ -169,4 +169,55 @@ $content-max-width-medium: 620px;
...
+  top: -2px;
+  margin: 0 5px;
+  border: 2px solid rgba(0, 0, 0, 0.5);
```
Comment (artisologic) [inline _sass/main/_home.scss]: with Sass you can do `rgba(black, 0.5)`

File: _sass/main/_page-indicators.scss [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+}
+
+.PageIndicators--center-bottom {
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
```
Comment (artisologic) [inline _sass/main/_page-indicators.scss]: What about using flex box to center align the dots?

File: _sass/main/_separator.scss [hunk 4]
```diff
@@ -1,9 +1,6 @@
-.SeparatorHorizontal {
-  width: 90%;
-  display: flex;
+%separator {
```
Comment (artisologic) [inline _sass/main/_separator.scss]: nitpicking, but for consistency let's call this `%Separator`

File: _sass/main/_separator.scss [hunk 5]
```diff
@@ -13,8 +10,50 @@
     display: inline-block;
     width: $size;
     height: $size;
-    background-color: $color-grey-light;
     position: relative;
+    background-color: currentColor;
+  }
+}
+
+%SeparatorHorizontal {
```
Comment (artisologic) [inline _sass/main/_separator.scss]: nitpicking, but for consistency let's call this `%HorizontalSeparator`

File: _sass/main/_separator.scss [hunk 6]
```diff
@@ -13,8 +10,50 @@
...
+  width: 90%;
+  display: flex;
+  border-top: 1px solid currentColor;
+
+  &:before, &:after {
     top: -4px;
   }
+}
+
+%SeparatorVertical {
```
Comment (artisologic) [inline _sass/main/_separator.scss]: nitpicking, but for consistency let's call this `%VerticalSeparator`

File: _sass/main/_utilities.scss [hunk 7]
```diff
@@ -1,15 +1,55 @@
...
+  font-size: $font-size-xsmall !important;;
+}
+
+.u-italic {
+  font-style: italic !important;;
+}
+
+.u-opacity-60 {
+  opacity: 0.6 !important;;
+  filter: alpha(opacity = 60) !important;;
```
Comment (artisologic) [inline _sass/main/_utilities.scss]: which browser doesn't support opacity these days?

File: _sass/main/_utilities.scss [hunk 8]
```diff
@@ -1,15 +1,55 @@
...
+}
+
+.u-sp-only {
```
Comment (artisologic) [inline _sass/main/_utilities.scss]: I'm confused by the naming here, what does `sp` refer too?

File: _sass/main/_utilities.scss [hunk 9]
```diff
@@ -1,15 +1,55 @@
...
+  @include media-query(small-device) {
+    display: none !important;;
+  }
+}
+
+.u-not-for-sp {
```
Comment (artisologic) [inline _sass/main/_utilities.scss]: Same here

File: _sass/main/_utilities.scss [hunk 10]
```diff
@@ -1,15 +1,55 @@
...
+  display: none !important;;
+
+  @include media-query(small-device) {
+    display: inherit !important;;
```
Comment (artisologic) [inline _sass/main/_utilities.scss]: This is tricky cause you are potentially setting the wrong display here.
Maybe the media query should be reversed to hide everywhere but small?

File: _sass/main/_utilities.scss [hunk 11]
```diff
@@ -1,15 +1,55 @@
...
+}
+
+.u-sp-only {
+  @include media-query(small-device) {
+    display: none !important;;
+  }
+}
+
+.u-not-for-sp {
+  display: none !important;;
```
Comment (artisologic) [inline _sass/main/_utilities.scss]: extra `;`

File: _sass/main/_utilities.scss [hunk 12]
```diff
@@ -1,15 +1,55 @@
...
+  @include media-query(small-device) {
+    display: inherit !important;;
+  }
+}
+
+.u-full-width {
+  width: 100% !important;;
+}
+
+.u-separator-full-width {
```
Comment (artisologic) [inline _sass/main/_utilities.scss]: What is this one for? Seems quite specific both in naming and in rules

File: index.html [hunk 13]
```diff
@@ -13,8 +15,14 @@
     </div>
 
     <div class="Home-content">
-        <div class="Home-dark">
+        <div class="Home-imageSection u-position-relative">
             <img class="Home-backgroundImage" src="assets/images/home_about_us.png">
+
+            <nav class="PageIndicators PageIndicators--center-bottom">
+                <a><span class="PageIndicators-dot"></span></a>
```
Comment (artisologic) [inline index.html]: Why not simply an `a` tag, do you need the extra element?

File: _sass/main/_separator.scss [hunk 14]
```diff
@@ -39,21 +40,26 @@
...
   color: $color-grey-light;
 }
 
 .VerticalSeparatorDark {
-  @extend %SeparatorVertical;
+  @extend %VerticalSeparator;
   color: $color-grey-dark;
+}
+
+.HorizontalSeparatorDark--full-width {
```
Comment (artisologic) [inline _sass/main/_separator.scss]: This one should probably be `.HorizontalSeparator--full-width` as it could be applied to a dark or light one.

Comment (jfidiles) [inline _sass/main/_separator.scss]: Very good point

**Comment by artisologic** [comment]
🎉

**Pull request merged**